### DISCUSSION TITLE
Randomizer State

### DISCUSSION THREAD

**Discussion by Naj-Rago** [phst-randomizer/ph-randomizer#471] [General]
Good Evening Peeps. My name is Naj Rago and i have been checking this page every ones in a while to see if theres anything new. I recently got interested to play the game again and was wondering how the current state of the rando is doin, And if i can help in any way. Im a regular to different Randomizers so im A good play tester and im also pretty good at spritework if needed. I hope everyone working on it is doing fine

**Comment by mike8699** [comment]
See https://github.com/phst-randomizer/ph-randomizer/discussions/488#discussioncomment-16213085
